client/lib/phone-validation/test/index.jsx

```diff
@@ -65,7 +65,7 @@ describe( 'Phone Validation Library', () => {
 	test( 'should pass a valid myanmar number with 9 digits after leading 9', () => {
 		assert.strictEqual( phoneValidation( '+959426123456' ).info, 'phone_number_valid' );
 	} );
-	test( 'should pass a valid Brazilian number starting with 49', () => {
```

Comment:
Strange. In theory, this should be a valid phone number. The `49` is the area code (it's known as `DDD` in Brazil, and it consists of a double-digit number). I've tested an actual real phone number from the `49` DDD and it didn't detect the right country/area either 🤔

Reply:
It's strange, it looks like it doesn't detect 8 digit BR phone-numbers as valid numbers from Brazil. I've tested v. `2.4.2` in Runkit and it has the same problem. I'll run the test locally to verify.

Reply:
Not sure. I did try with a 3rd-party service and got the same result, though:
https://freetools.textmagic.com/phone-validator

It's worth noting that it's supposed to only return a valid result when it's a mobile phone number, not a landline.

Reply:
Oh I see, it only works for mobile phone numbers, it seems ([according to its README](https://github.com/AfterShip/phone/blob/master/README.md))? 🤯

If that's the case, then it's okay. Mobile numbers do have a greater length than landlines in BR. So I guess we're fine.